cypress/support/fragments/data-export/dataExportResults.js

```diff
@@ -0,0 +1,11 @@
+export default {
+  verifyQuickExportResult() {
+    cy.get('[data-row-index="row-0"]').then(elem => {
```

Comment:
if you have issues with implementation through interactor - add details into TODO.